HOWTO-bn.md

```diff
@@ -2,12 +2,12 @@
 
 Free-Programming-Books রিপোজটরি তে স্বাগতম! আমরা নবাগত কন্ট্রিবিউটরস দের স্বাগতম জানাই; এমনকি যারা গিটহাবে এই প্রথম কোন "পুল রিকোয়েস্ট" তৈরি কয়েছেন। যদি আপনি তাদের একজন হয়ে থাকেন তাহলে নিচের রিসোর্স গুলো আপনার কাজে লাগতে পারেঃ
 
-* [পুল রিকোয়েস্ট কি?](https://help.github.com/articles/about-pull-requests/)
-* [কিভাবে পুল রিকোয়েস্ট দিব](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
-* [গিটহাব হ্যালো ওয়ার্ল্ড](https://guides.github.com/activities/hello-world/)
-* [ইউটিউব - নতুনদের জন্য গিটহাব](https://www.youtube.com/watch?v=0fKg7e37bQE)
-* [ইউটিউব - কিভাবে একটি গিটহাব রিপোজিটরি ফোর্ক করবেন এবং পুল রিকোয়েস্ট সাবমিট করবেন](https://www.youtube.com/watch?v=G1I3HF4YWEw)
-* [ইউটিউব - মার্কডাউন ক্র্যাশ কোর্স ](https://www.youtube.com/watch?v=HUBNt18RFbo)
+* [:us: পুল রিকোয়েস্ট কি?](https://help.github.com/articles/about-pull-requests/) *(in english)*
+* [:us: কিভাবে পুল রিকোয়েস্ট দিব](https://docs.github.com/en/free-pro-team@latest/github/ collaborating-with-issues-and-pull-requests/creating-a-pull-request) *(in english)*
```

Comment:
There are an extra space in the middle of URL

Reply:
Fixed at https://github.com/EbookFoundation/free-programming-books/commit/987e0b733c8265cd743041a5fea51c6f3239346e